go.mod

```diff
@@ -104,7 +104,7 @@ require (
 	github.com/talos-systems/grpc-proxy v0.3.0
 	github.com/talos-systems/net v0.3.2
 	github.com/talos-systems/siderolink v0.1.2
-	github.com/talos-systems/talos/pkg/machinery v1.1.0-alpha.2
```

Comment:
guess we missed this for beta.0 :sweat_smile:

Reply:
yeah, I realized that after cutting a release :facepalm: